File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PromoteFromInternalToBetaTask.java
package com.dipien.google.play.api.task;

import com.dipien.google.play.api.App;
import com.dipien.google.play.api.PublishingService;

public class PromoteFromInternalToBetaTask extends BaseTask {
	
	public PromoteFromInternalToBetaTask() {
		setDescription("Promote a current internal to beta");
	}

	@Override
	protected void onExecute(App app) {
		new PublishingService().promoteFromInternalToBeta(app);
	}

}

File: google-play-api-plugin/src/main/java/com/dipien/google/play/api/task/PromoteFromInternalToBetaTask.kt
package com.dipien.google.play.api.task

import com.dipien.google.play.api.App
import com.dipien.google.play.api.PublishingService

class PromoteFromInternalToBetaTask : BaseTask() {
    override fun onExecute(app: App) {
        PublishingService().promoteFromInternalToBeta(app)
    }

    init {
        description = "Promote a current internal to beta"
    }
}
